Docs/source/usage/workflows/python_extend.rst

```diff
@@ -134,7 +134,7 @@ This example accesses the :math:`E_x(x,y,z)` field at level 0 after every time s
        warpx = sim.extension.warpx
 
        # data access
-       E_x_mf = warpx.multifab(f"Efield_fp[x][level=0]")
+       E_x_mf = warpx.multifab(f"E_fp[x][level=0]")
```

Comment:
This should be changed to `E_x_mf = warpx.multifab("E_fp", dir=0, level=0)` in a later PR if not already fixed.